control-plane/agents/src/bin/core/controller/scheduling/resources/mod.rs

```diff
@@ -110,6 +115,7 @@ impl ReplicaItem {
         child_state: Option<Child>,
         child_spec: Option<NexusChild>,
         child_info: Option<ChildInfo>,
+        vg_replica_count_on_pool: Option<u64>,
```

Comment:
```suggestion
        vg_replicas_on_pool: Option<u64>,
```
match the struct name for consistency

Reply:
Right. Missed this. Thanks.